src/fontra/views/editor/panel-selection-info.js

```diff
@@ -141,18 +141,18 @@ export default class SelectionInfoPanel extends Panel {
     if (
       selectedGlyphInfo?.isUndefined &&
       selectedGlyphInfo.character &&
-      !unicodes.length
+      !codePoints.length
     ) {
       // Glyph does not yet exist in the font, but we can grab the unicode from
       // selectedGlyphInfo.character anyway
-      unicodes = [selectedGlyphInfo.character.codePointAt(0)];
+      codePoints = [selectedGlyphInfo.character.codePointAt(0)];
     }
 
-    const unicodesStr = makeUnicodesString(unicodes);
-    let baseUnicodesStr;
-    if (glyphName && !unicodes.length) {
+    const codePointsStr = makeCodePointsString(codePoints);
```

Comment:
I'm a little bot torn about the changes here: we're producing a string that is shown to the user with a "Unicode:" label (which I think is good). Same for `baseUnicodesStr`. Let's discuss with a fresh mind.

Reply:
@justvanrossum 
The question is, when do we want or have to use the term unicode and when is codePoint better? Is a unicode a string representation of a codePoint? The decimal value 65 is a codePoint, but converted to Hex and displayed as a string like '0041' is a unicode?
65 = codePoint
'0041' = unicode

Reply:
I think of `U+0041` as a "unicode code point", so in the end it doesn't matter all that much how we name the variables here :) Let's keep your change, I think it's fine.